checkov/common/graph/checks_infra/registry.py

```diff
@@ -28,7 +28,18 @@ def run_checks(
     ) -> dict[BaseGraphCheck, list[_CheckResult]]:
 
         check_results: "dict[BaseGraphCheck, list[_CheckResult]]" = {}
-        checks_to_run = [c for c in self.checks if runner_filter.should_run_check(c, report_type=report_type)]
+        graph_sources_path = list(
+            set([graph_connector.nodes.get(node).get("file_path_")
+                 for node in graph_connector.nodes if graph_connector.nodes.get(node) and
+                 graph_connector.nodes.get(node).get("file_path_")])
+        )
```

Comment:
1. This disables the check for all files, so not the best way to put this
2. To iterate over the node data, you can just do `graph_connector.nodes(data=True)`. There are examples in checkov for this
3. Please use `CustomAttributes.FILE_PATH` instead of the hardcoded `file_path_`
@pazbechor

Reply:
Thanks :) 
Done